Assets/Scripts/Content/QuestData.cs

```diff
@@ -1179,8 +1179,8 @@ override public string ToString()
     public class Puzzle : Event
     {
         new public static string type = "Puzzle";
-        public string puzzleClass = "slide";
-        public string skill = "{observation}";
+        public StringKey puzzleClass = new StringKey("val", "PUZZLE_SLIDE_CLASS");
```

Comment:
This was my idea. to Key all 3 puzzle types. I will try to do same thing with traits and other magic strings. Tried only this in order to know  your oppinion or if it would crash somehow